tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -228,8 +228,13 @@ def analyze_latency_dump(array):
 
 def analyze_throughput_dump(array):
     failed = 1
+    array_len = len(array)
 
     for i in range(18, 36):
+        if i >= array_len:
+            print("Array len is ", array_len, ", expecting >= ", i)
+            print(array)
+            return failed
```

Comment:
I think you might need to do similar things here https://github.com/awslabs/s2n/blame/master/tests/integration/s2n_dynamic_record_size_test.py#L216. Looks like the tcpdump output check need update after we introduced TLS1.3.

Reply:
so far the tests that have been failing in travis seem to fall on analyze_throughput_dump over analyze_latency_dump, we could add more print conditions if we find it's failing there